lib/cli-config.js

```diff
@@ -68,14 +71,34 @@ exports.getContent = function(config, directory) {
         config = path.basename(config);
         ext = path.extname(configPath);
 
-        if (ext === '.js' || ext === '.json') {
+        if (ext === '.js') {
             content = require(configPath);
         } else {
-            content = JSON.parse(
-                stripJSONComments(
-                    fs.readFileSync(configPath, 'utf8')
-                )
-            );
+            data = fs.readFileSync(configPath, 'utf8');
+
+            if (ext === '.json') {
+                content = parseJson(stripJSONComments(data));
+            } else if (ext === '.yaml') {
+                content = yaml.safeLoad(data);
+            } else {
+                // try both JSON and YAML
+
+                try {
+                    content = parseJson(stripJSONComments(data));
+                } catch (jsonError) {
+                    try {
+                        content = yaml.safeLoad(data);
+                    } catch (yamlError) {
+                        if (stripJSONComments(data).trim()[0] === '{') {
+                            // the intention was probably JSON
+                            throw jsonError;
+                        } else {
+                            // assume the intention was YAML
+                            throw yamlError;
+                        }
+                    }
+                }
+            }
```

Comment:
What you think about determining format with this check?
`if (=== '{') then try to parseJson else try to parseYaml.`